src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
> Works for me. Is this a) also the suggested solution for dropdowns, or do b) we keep the dividers there? If a) do we need a special group for this (since there we do not have a title or such)? If b) I think then we can simplify the dividers further since they do not need a title or such.

We still need the Divider PR. I would prefer b). But we definitely need a title. Think of the current "Repository" Dropdown in the main menu. It comes with a "Last Visited" Divider. I am not sure if the Administration submenu will still be considered being a Dropdown, but again we would have Dividers with titles.